server/api/student/student.model.js

```diff
@@ -43,6 +45,18 @@ const studentSchema = new Schema({
         type: Number,
         min: 0,
         max: 100
+    },
+    social: {
+        enum:socialgrades,
```

Comment:
Does it work with enum AND type number? isn't type number sufficient?

Reply:
Right  the type should be enough with the min,max attributes because its numeric.
I deleted the enum reference for the next commit.